oviewer/oviewer.go

```diff
@@ -294,6 +295,18 @@ func Open(fileNames ...string) (*Root, error) {
 	return openFiles(fileNames)
 }
 
+func NewRoot(read io.Reader) (*Root, error) {
+	m, err := NewDocument()
+	if err != nil {
+		return nil, err
+	}
+
+	if err = m.ReadAll(io.NopCloser(read)); err != nil {
```

Comment:
Changed NopCloser so that it is not needed.
Can you fix it to just pass the read?